app/services/dashboard/participants.rb

```diff
@@ -21,11 +23,11 @@ def completed_induction_count
     end
 
     def currently_training_count
-      ects.size
+      (currently_training_ects + orphan_ects + active_mentors).size
```

Comment:
Since we want the mentors to be displayed in the "Currently training" (unless they are no longer training), then it's natural to include them in the currently training count. 

@frankieroberto Is that correct?